### ISSUE TITLE
Title for leaflet legend

### ISSUE THREAD

**Issue by jukos** [tomchadwin/qgis2web#844]
Hello,
**it is possible to integrate text/title within the legend completly above the layer/checkboxes?**
With a small trick a can set groupnames for different layers: html within particular layernames within qgis. Unfortunately I can not integrate text with this trick at the top of the legend. 
Any ideas? 
![000872](https://user-images.githubusercontent.com/31866672/70902063-45b60d80-1ffc-11ea-906c-63631cfdcb49.jpg)

**Comment by riccardoklinger** [comment]
I dont think this can be done easily within the plugin. as a workaround, jquery comes in handy:
add the src in the scipts-section of the index.html:
`         <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>`
and later at the end of the script section below the  control creation add:
`        $(".leaflet-control-layers-overlays").prepend("<h2><label>Legende</label></h2>");`
<img width="303" alt="grafik" src="https://user-images.githubusercontent.com/7377897/71326707-18d39000-24ff-11ea-99f1-23b1a729a649.png">

**Comment by jukos** [comment]
Perfect! Thank you very much.
Do you know, if I write a legal notice for use this script?

**Comment by riccardoklinger** [comment]
I didn't get it, say it in German please

**Comment by jukos** [comment]
Ich versuche immer darauf zu achten, ob es eine Pflicht gibt eine Quelle und/oder Lizenz zu nennen.
Bin inzwischen fündig geworden:
https://github.com/googleapis/googleapis/blob/master/LICENSE
Ich weiß zwar nicht genau, ob das in diesem Fall zutrifft, aber im Zweifelsfalle verlinke ich auf diese und auf die Lizenzseite. Es gibt genügend (Juristen)-Jäger und ich möchte kein Wild sein. Leider muss man in Deutschland so denken. Und falls jemand etwas für Kunden herstellt sowieso.

Grüße, Jürgen

**Comment by riccardoklinger** [comment]
HI Jürgen, can we close this though?

**Comment by jukos** [comment]
Yes. Thank you!
